docs/release-notes/RELEASE-NOTES-3.1.0.md

```diff
@@ -8,7 +8,13 @@ It:
   functionality is being built on data frames instead of using the control
   frames.
 * Enhances the HTTP endpoint for channel creation transaction to accept an optional list of delegates.
-* Introduces State Channel fsm-assisted solo-close and slash, and continous chain monitoring
+* Introduces State Channel fsm-assisted solo-close and slash, and continuous chain monitoring.
```

Comment:
@uwiger I kept sentence below "This release is backward compatible with the previous v3.0.1 release."